### PR TITLE
Exclusion report intro

### PR DESCRIPTION
**Pull request by amichard**
#1339 #1340 #1341 

Added Intro page for Exclusion Report. Also, added drop-down for some columns

Changelog:
- Added Intro page
- Added new model Transaction Type
- Created operational script for Transaction Type
- Updated dockerized for new operational script
- Added EXCLUSION_REPORTS config for backend
- Moved some front-end components into the app side so it's much easier to be reused

### CODE REVIEW

**Comment by amichard** [comment]
[Exclusion Report | Introduction page |2|](https://trello.com/c/75qadlhA/1524-exclusion-report-introduction-page-2)

**Comment by amichard** [comment]
[Exclusion Report | Transaction Type |3|](https://trello.com/c/qVp55ljn/1556-exclusion-report-transaction-type-3)

**Comment by amichard** [comment]
[Exclusion Report | Fuel Type |1|](https://trello.com/c/gyoynfCg/1557-exclusion-report-fuel-type-1)

**Comment by amichard** [comment]
[Exclusion Report | Legal Name and Address of Transaction Partner |1|](https://trello.com/c/UL0nnVph/1558-exclusion-report-legal-name-and-address-of-transaction-partner-1)

File: backend/api/models/TransactionType.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+class TransactionType(Auditable, DisplayOrder, EffectiveDates):
+    """
+    Contains a list of possible transaction types for the
+    Exclusion Agreement Report
+    """
+    the_type = models.CharField(
+        max_length=25,
+        unique=True,
+        db_comment="Transfacion types for Exclusion Agreement Report."
```
Comment (klefler) [inline backend/api/models/TransactionType.py]: Minor thing... Transaction is spelled incorrectly.

Comment (amichard) [reply backend/api/models/TransactionType.py]: lol good catch. Thanks Kristin!

**Pull request merged**